keyboards/nopunin10did/railroad/info.json

```
@@ -0,0 +1,124 @@
+                {"y":4.25, "x":9.25, "label":"Shift / Del", "w":2.75},
+                
+                {"y":0, "x":10.25, "label":"F1"}, 
+                {"y":0, "x":11.25, "label":"F5"},        
+                {"y":0, "x":12.25, "label":"F9"}, 
+                {"y":1, "x":10.25, "label":"F2"}, 
+                {"y":1, "x":11.25, "label":"F6"},        
+                {"y":1, "x":12.25, "label":"F10"},
+                {"y":2, "x":10.25, "label":"F3"}, 
+                {"y":2, "x":11.25, "label":"F7"},        
+                {"y":2, "x":12.25, "label":"F11"}, 
+                {"y":3, "x":10.25, "label":"F4"}, 
+                {"y":3, "x":11.25, "label":"F8"},        
+                {"y":3, "x":12.25, "label":"F12"},
+                
+                {"y":0, "x":13.5, "label":"="},
+                {"y":0, "x":14.5, "label":"6"},
+                {"y":0, "x":15.5, "label":"7"},
+                {"y":0, "x":16.5, "label":"8"},
+                {"y":0, "x":17.5, "label":"9"},
+                {"y":0, "x":18.5, "label":"0"},
+                {"y":0, "x":19.5, "label":"Backspace", "w":2},
+                {"y":0, "x":21.5, "label":"Ins"},
+                {"y":0, "x":22.5, "label":"Esc"},
+                
+                {"y":1, "x":13.5, "label":"]"},
+                {"y":1, "x":14.5, "label":"Y"},
+                {"y":1, "x":15.5, "label":"U"},
+                {"y":1, "x":16.5, "label":"I"},
+                {"y":1, "x":17.5, "label":"O"},
+                {"y":1, "x":18.5, "label":"P"},
+                {"y":1, "x":19.5, "label":"\\", "w":1.5},
+                {"y":1, "x":21,   "label":"Del"},
+                
+                {"y":2, "x":13.5, "label":"Home"},
+                {"y":2, "x":14.5, "label":"H"},
+                {"y":2, "x":15.5, "label":"J"},
+                {"y":2, "x":16.5, "label":"K"},
+                {"y":2, "x":17.5, "label":"L"},
+                {"y":2, "x":18.5, "label":";"},
+                {"y":2, "x":19.5, "label":"'"},
+                {"y":2, "x":20.5, "label":"Return", "w":1.75},
+                
+                {"y":1, "x":22.25, "label":"ISO Enter", "h":2, "w":1.25},
+                
+                {"y":3, "x":13.5,    "label":"End"},
+                {"y":3, "x":14.5,    "label":"N"},
+                {"y":3, "x":15.5,    "label":"M"},
+                {"y":3, "x":16.5,    "label":","},
+                {"y":3, "x":17.5,    "label":"."},
+                {"y":3, "x":18.5,    "label":"/"},
+                {"y":3, "x":19.5,    "label":"Shift", "w":1.75},
+                {"y":3.25, "x":21.5, "label":"Up"},
+                
+                {"y":4.25, "x":12,    "label":"LShftAlt / Calc", "w":2.25},
+                {"y":4.25, "x":14.25, "label":"Space", "w":1.75},
+                {"y":4.25, "x":16,    "label":"UK #~"},
+                {"y":4, "x":17.25,    "label":"AltGr / Home", "w":1.5},
+                {"y":4, "x":17.25,    "label":"Ctrl / End", "w":1.5},
```

Your Right AltGr and Right Ctrl keys are actually occupying the same space here.

```suggestion
                {"y":4, "x":18.75,    "label":"Ctrl / End", "w":1.5},
```

---

Good catch!